library/common/config/config.cc

```diff
@@ -143,7 +143,7 @@ R"(- &enable_drain_post_dns_refresh false
 !ignore admin_interface_defs: &admin_interface
     address:
       socket_address:
-        address: 127.0.0.1
```

Comment:
@snowp @mattklein123 the IPv4 address in here stopped working after we started enforcing the use of IPv6 addresses on Android. Is moving to IPv6 address an acceptable solution here (it does seem to work)?

Reply:
This is used for when we enable Envoy admin interface so that we are able to do things such as `curl localhost:9901/stats` in the terminal to get the state of EM stats.